vscode-antimony/src/server/stibium/stibium/analysis.py

```diff
@@ -225,6 +231,8 @@ def check_parse_tree(self, root, scope):
                     self.process_is_assignment(node, scope)
                 elif type(node.get_stmt()) == Assignment:

```

Comment:
Use isinstance

Reply:
> Use isinstance

We should create another PR for all this type of problem, because it has been the way it is.